index.js

```diff
@@ -332,6 +364,11 @@ module.exports.stderr = async (...args) => {
 	return stderr;
 };
 
+module.exports.all = async (...args) => {
+	const {all} = await module.exports(...args);
+	return all;
+};
```

Comment:
I don't want a shortcut method for `all`. The existing ones only exist because we didn't have destructuring before.